lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,609 @@
+-- The tazer is no longer a dependancy :^) --
+AllowedWake = { -- these conditions can be removed when a friend presses +use
```

Comment:
After looking through each file, this does not seem to be referenced outside of this file.
This also does not need to be a global, slipped through the first time.